bubblesub/ui/model/events.py

```diff
@@ -131,6 +131,12 @@ def display(self, sub: AssEvent) -> T.Any:
             else "-"
         )
 
+    def read(self, sub: AssEvent) -> T.Any:
+        pass
```

Comment:
Same as above, not implemented means not implemented, and it should stay this way.

Consider disabling this warning for these classes, including for the `_get_paint_cache_key` overrides. I don't have any strong opinions here.

Reply:
Better to be explicit here and implement the abstract methods imho

Reply:
OK but let's raise a not implemented error as well, as these are really supposed to be not implemented (I mean for example you shouldn't be able to write values to the CPS column)

Reply:
Sure, I agree